Sources/Lua/LuaFoundation.swift

```diff
@@ -11,32 +11,38 @@ import CLua
 /// This enum exists because CoreFoundation contains many encodings not supported by `String.Encoding`.
 public enum LuaStringEncoding {
     case stringEncoding(String.Encoding)
+#if !os(Linux)
```

Comment:
These could maybe be `canImport(CoreFoundation)` instead?

Reply:
Yeah. That sounds much cleaner. Certainly the conditional checks on Linux are pretty blunt.